includes/polling/functions.inc.php

```diff
@@ -445,7 +445,10 @@ function location_to_latlng($device) {
     if (!empty($device_location)) {
         $new_device_location = preg_replace("/ /","+",$device_location);
         // We have a location string for the device.
-        $loc = dbFetchRow("SELECT `lat`,`lng` FROM `locations` WHERE `location`=? LIMIT 1", array($device_location));
+        $loc = parse_location($device_location);
```

Comment:
Because the current version needs a space (format: "[0.12345, 0.12345]"), it gives a problem with line 446 in this file, as we try to parse a line after calling: `preg_replace("/ /","+",$device_location);`